src/test/java/StringTest.java

```diff
@@ -0,0 +1,49 @@
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("문자열 자료형 관련 테스트")
+public class StringTest {
+
+    @DisplayName("콤마를 기준으로 숫자를 분리한다.")
+    @Test
+    void splitNumber() {
+        // given
+        String numberOneAndTwo = "1,2";
+        String numberOne = "1";
+
+        // then
+        String[] splitOneAndTwoNumber = numberOneAndTwo.split(",");
+        String[] splitOneNumber = numberOne.split(",");
+
+        // then
+        Assertions.assertThat(splitOneAndTwoNumber).contains("1");
```

Comment:
```suggestion
        Assertions.assertThat(splitOneAndTwoNumber). containsExactly("1", "2");
```
2까지 포함되었는지 확인해야 하지 않을까요?